prompts.go

```diff
@@ -65,3 +72,64 @@ func PromptOrgAndProject(scope []string) bool {
 	}
 	return promptConfirm
 }
+
+func PromptUrlNG() bool {
+	promptConfirm := false
+	re := regexp.MustCompile(`https:\/\/app\.harness\.io/ng/#/account/([a-zA-Z0-9]+)/.*/orgs/([a-zA-Z0-9_]+)/projects/([a-zA-Z0-9_]+)/.*`)
```

Comment:
This will only work for Prod. It will not work in the case of Dev/QA/Prod3. We also want to support on-premise in the future. 
We are also prompting for project/org. In the case of account-level entity migration, we do not need to prompt for projects.